samples/test/test.samples.auth.js

```diff
@@ -41,7 +41,8 @@ describe('Auth samples', () => {
     const realPath = path.resolve('jwt.keys.json');
     const exists = fs.existsSync(realPath);
     if (!exists) {
-      fs.symlinkSync(fakePath, realPath);
+      const contents = fs.readFileSync(fakePath);
```

Comment:
curious why we needed to move away from `symlinkSync` to writing a file?

Reply:
See my reply [here](https://github.com/googleapis/google-api-nodejs-client/issues/2377#issuecomment-702694454).
Sorry for spanning the changes in 2 different PRs...